pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -359,7 +361,11 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	log.Printf("Found worker nodes security group %s", workerSgId)
 
 	log.Print("Ensuring pods security group")
-	podSgId, err := ensureOpenStackSg(client, generateName("kuryr-pods-security-group", clusterID), tag)
+	var podSgId string
+	podSgId, err = ensureOpenStackSg(client, generateName("kuryr-pods-security-group", clusterID), tag)
```

Comment:
again, why introduce an explicit `var` here (and below)? Adding the `err != nil` check makes sense, but adding the `var` and removing the `:=` doesn't

Reply:
fixed in updated patchset